app/views/assistants/_assistant.html.erb

```diff
@@ -44,13 +44,14 @@
 
       <%= link_to new_assistant_message_path(assistant), class: "inline-flex", data: { role: "pencil" } do %>
         <%= icon "pencil-square",
-          variant: :micro,
+          variant: :outline,
           size: 18,
-          class: "text-gray-400 cursor-pointer group-hover:visible hover:text-white invisible relationship:visible relationship:text-white",
+          class: "text-gray-950 dark:text-gray-100 cursor-pointer group-hover:visible invisible 
+                  relationship:visible relationship:text-gray-950 dark:relationship:text-gray-100",
           'data-controller': "nested-pointer",
           title: "New"
```

Comment:
Did you think the outline version looked better? At this small size, I thought the micro looked a bit nicer. I thought detail of the "outline" version got a bit lost at this size. But maybe you did it for consistency since, now that I'm looking at things, it looks like we might have outline icons everywhere else.

Reply:
Let's keep the change you made!

Reply:
Let's keep the change you made! To be honest, I don't love the heroicon pack. The other pencil icon (the one for edit) is particularly weak at this small size. But it's a bigger project to try and switch that out and I think it's good enough to make a solid product. At some point we'll introduce a few alternate themes, and I think that's the time to revisit icons in a bigger way.